kinesis_video_msgs/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <package format="3">
   <name>kinesis_video_msgs</name>
-  <version>0.0.0</version>
+  <version>3.0.0</version>
```

Comment:
Changelog should have v3 in it

Reply:
The changelog is imported from the ROS1 version so that's intentional - it will be updated for 3.0.0 as part of our bloom workflow.